bridges/snowbridge/pallets/outbound-queue-v2/src/benchmarking.rs

```diff
@@ -19,45 +19,77 @@ use crate::Pallet as OutboundQueue;
 mod benchmarks {
 	use super::*;
 
-	/// Benchmark for processing a message.
-	#[benchmark]
-	fn do_process_message() -> Result<(), BenchmarkError> {
-		let enqueued_message = Message {
+	/// Build `Upgrade` message with `MaxMessagePayloadSize`, in the worst-case.
+	fn build_message_in_worst_case<T: Config>() -> (Message, OutboundMessage) {
```

Comment:
```suggestion
	fn build_worst_case_message<T: Config>() -> (Message, OutboundMessage) {
```

Reply:
Its a given that we need to benchmark the worst case, in which case we can surely just name this:

```suggestion
	fn build_message<T: Config>() -> (Message, OutboundMessage) {
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/61/commits/4ba351995ead17631bd3c5e7f5edaa145cbdf8db